.gitattributes

```diff
@@ -6,6 +6,9 @@ gradlew						text eol=lf
 */src/*/assets/**/*expected*.yml		text eol=lf
 reporter/src/funTest/assets/*-expected-*	text eol=lf
 
+# Use Unix line endings for Freemarker templates for consistency across platforms.
+reporter/src/main/resources/template.notice/**
```

Comment:
Wouldn't `*.ftlh` instead of `**` do here and also be clearer WRT the scope?

Reply:
Done, I also made it a bit shorter to not have to reformat the other lines. Also, I forgot to add `text eol=lf`.